applications/nrf_desktop/README.rst

```diff
@@ -1102,7 +1102,8 @@ nRF52840 Gaming Mouse (nrf52840gmouse_nrf52840)
         * Bluetooth is configured to use Nordic's SoftDevice link layer.
 
       * The configuration with the B0 bootloader is set as default.
-      * The board supports debug :ref:`nrf_desktop_bluetooth_guide_fast_pair` configuration (:file:`prj_fast_pair.conf`).
+      * The board supports debug (:file:`prj_fast_pair.conf`) and release (:file:`prj_release_fast_pair.conf`) :ref:`nrf_desktop_bluetooth_guide_fast_pair` configurations.
```

Comment:
https://github.com/nrfconnect/sdk-nrf/pull/11173#discussion_r1193918744

Please move the release note comment regarding the prj_mcuboot configuration for nRF52840 DK

Reply:
Added here